internal/cmd/new/new_test.go

```diff
@@ -0,0 +1,69 @@
+package new
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestNew(t *testing.T) {
+	dir := os.TempDir()
+	err := os.Chdir(dir)
+	if err != nil {
+		t.Fatal("error changing to tmp dir")
```

Comment:
Just to be in line with other tests; should we use `require` from `testify` here instead of `t.Fatal`?

EDIT: there are already tests for the new command in `internal/genny/newapp/api`. Would it make sense to move these tests from here to there?

Reply:
I was hoping to provide a higher level of tests, maybe I should add tests there as well. Will do it.

Reply:
I edited my comment, I was refering to the wrong test packages. The package `internal/cmd/internal/integration` seems to be the correct place for high level tests.

It only seems like I've accidentally broken the existing few tests there. In a former PR I deleted this file: https://github.com/gobuffalo/cli/blob/c56e8d08bbead0273da5202421c2c0da21b0c9cb/internal/cmd/internal/integration/integration_test.go probably by accident 😓

Reply:
yes. Agree, I can take care of that. Will have those added back in this PR.

Reply:
Now, I'm hesitant of a couple of things there, I think tests for the `new` command should be next to it, feels like the way Go does it.